### DISCUSSION TITLE
Cannot type in terminal in  remote version

### DISCUSSION THREAD

**Discussion by kasraTM** [community/community#39692] [Codespaces]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/114722176/203142567-270a2e85-e25a-4c86-a1bc-0e9dd24b9720.jpg)

Hello world,

I am facing a weird problem, I use vs code remote on 2 devices, 1- on a windows operated system,  2- on a mac,  but on both I use the remote version.   On the windows one the terminal and the code-space works perfectly,  but on my mac, the terminal is blocked, as you can see in the picture, it is greyish, and does not allow me to type in any command,  it was not like this before,  any solution ?

**Comment by david-engelmann** [comment]
You will probably get the best results if you submit this as a bug issue in the [Vscode remote release repository](https://github.com/microsoft/vscode-remote-release/issues)!
